streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -92,7 +92,6 @@
     private final Sensor closeTaskSensor;
     private final Sensor processLatencySensor;
     private final Sensor punctuateLatencySensor;
-    private final Sensor commitSensor;
```

Comment:
This is an open question: we don't want to remove this sensor however it was unclear to me how to handle this metric after we split "task committing" into three steps (prepareCommit; taskManager#commit; postCommit).

Reply:
Frankly, I have no good idea atm... Also, if we change metrics, we need to update the KIP and it's getting more complicated. If possible, I would prefer to not change any metric, but not sure if it is possible...

Reply:
I actually think that we can remove this DEBUG-level per-task commit metrics, since we already have the INFO-level per-thread commit metric and this one does not provide much more additional information?